frontend/src/pages/Chat/ChatRoom.tsx

```diff
@@ -0,0 +1,239 @@
+import { FormEvent, useContext, useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { LogoutIcon, SendIcon } from '../../assets/icons/icons';
+import ChatList from '../../components/Chat/ChatList';
+import ChatMessage from '../../components/Chat/ChatMessage';
+import Loading from '../../components/commons/Loading';
+import Modal from '../../components/commons/Modal/Modal';
+import CustomInput from '../../components/CustomInput';
+import PageHeader from '../../components/PageHeader/PageHeader';
+import { UserInfoContext } from '../../context/UserInfoContext';
+import { useModal } from '../../hooks/useModal';
+import useSocket from '../../hooks/useSocket';
+import { remote } from '../../lib/api';
+import { useNavigate, usePathParams } from '../../lib/Router';
+import colors from '../../styles/colors';
+import { textMedium, textSmall } from '../../styles/fonts';
+import { ChatRoomInfo } from '../../types/chat';
+
+function Chat() {
+  const [message, setMessage] = useState('');
+  const { chatId } = usePathParams();
+  const [chatRoomInfo, setChatRoomInfo] = useState<ChatRoomInfo>(null!);
+  const userInfo = useContext(UserInfoContext);
+  const [partner, setPartner] = useState<string>('partner');
+  const { isModalOpen, openModal, closeModal } = useModal();
+  const navigate = useNavigate();
+
+  // const { data: chatRoomInfo } = useQuery<ChatRoomInfo>(
+  //   ['chat', chatId],
```

Comment:
```suggestion
  //   ['chat' + chatId, chatId],
```

이런 식으로 chatId 별로 키 값을 제공하면 방마다 캐시가 다르게 적용되기는 할거에요 🚀

Reply:
이것도 한번 테스트해보겠습니다~